src/app/modules/ix-forms/components/ix-ip-input-with-netmask/ix-ip-input-with-netmask.component.ts

```diff
@@ -71,8 +71,7 @@ export class IxIpInputWithNetmaskComponent implements ControlValueAccessor {
     this.onChange(value);
   }
 
-  private setAddressAndNetmask(ipWithNetmask: string): void {
-    ipWithNetmask = ipWithNetmask || '';
+  private setAddressAndNetmask(ipWithNetmask = ''): void {
```

Comment:
Previously if you got a value from somewhere that you didn't know is null, you could call the method with that value safely. Now, it's possible to call the method without a  net mask at all. Maybe a different method is better. `setEmptyAddressAndNetmask(): void { this.setAddressAndNetMask(''); }` and the check can be performed outside the method. As is, you could assume that the method just sets the correct values if you just call it without any values.

Reply:
It's only called in one place where this is not an issue.